dlp/system-test/inspect.test.js

```diff
@@ -520,4 +520,25 @@ describe('inspect', () => {
     }
     assert.include(output, 'INVALID_ARGUMENT');
   });
+
+  // dlp_inspect_string_multiple_rules
+  it('should inspect using multiple rules', () => {
+    const output = execSync(
+      `node inspectStringMultipleRules.js ${projectId} "patient: Jane Doe"`
```

Comment:
Please see https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/dlp/snippets/src/test/java/dlp/snippets/InspectTests.java
Tests starting with 'testInspectStringMultipleRules'. We should add all the scenarios.

Reply:
Updated the tests, thanks